spec/spec_helper.rb

```diff
@@ -168,10 +168,3 @@ def delete_test_indices
 def get_message_part (mail, content_type)
```

Comment:
This can be removed entirely, in favor of `email.html_part` or `email.text_part`.